src/components/Admin/AllDevices.vue

```diff
@@ -1,162 +1,191 @@
                 <v-card-actions>
                   <v-spacer />
                   <v-btn
                     color="#600000"
                     text
-                    @click="close"
+                    @click="closeDelete"
                   >
                     Cancel
                   </v-btn>
                   <v-btn
                     color="#600000"
                     text
-                    @click="save"
+                    @click="deleteItemConfirm"
                   >
-                    Save
+                    OK
                   </v-btn>
+                  <v-spacer />
                 </v-card-actions>
               </v-card>
             </v-dialog>
-          </v-toolbar>
-          <v-dialog
-            v-model="dialogDelete"
-            max-width="500px"
-          >
-            <v-card>
-              <v-card-title class="text-h5">
-                Are you sure you want to delete this item?
-              </v-card-title>
-              <v-card-actions>
-                <v-spacer />
-                <v-btn
-                  color="#600000"
-                  text
-                  @click="closeDelete"
-                >
-                  Cancel
-                </v-btn>
-                <v-btn
-                  color="#600000"
-                  text
-                  @click="deleteItemConfirm"
-                >
-                  OK
-                </v-btn>
-                <v-spacer />
-              </v-card-actions>
-            </v-card>
-          </v-dialog>
-        </template>
-        <template v-slot:[`item.actions`]="{ item }">
-          <v-icon
-            small
-            class="mr-2"
-            @click="editItem(item)"
-          >
-            mdi-pencil
-          </v-icon>
-          <v-icon
-            small
-            @click="deleteItem(item)"
-          >
-            mdi-delete
-          </v-icon>
-        </template>
-      </v-data-table>
-    </v-card>
+          </template>
+          <template v-slot:[`item.actions`]="{ item }">
+            <v-icon
+              small
+              class="mr-2"
+              @click="editItem(item)"
+            >
+              mdi-pencil
+            </v-icon>
+            <v-icon
+              small
+              @click="deleteItem(item)"
+            >
+              mdi-delete
+            </v-icon>
+          </template>
+        </v-data-table>
+      </v-card>
+    </v-layout>
   </v-container>
 </template>
 
 <script>
+// eslint-disable-next-line import/no-duplicates
 import { getCollection } from '../../firebase/techCenterCheckout.Data';
+// eslint-disable-next-line import/no-duplicates
+import { deleteDevice } from '../../firebase/techCenterCheckout.Data';
+// eslint-disable-next-line import/no-duplicates
+import { updateDevice } from '../../firebase/techCenterCheckout.Data';
+// eslint-disable-next-line import/no-duplicates
+import { addDevice } from '../../firebase/techCenterCheckout.Data';
```

Comment:
If you see an eslint warning, it's probably because you're doing something the wrong way. Don't just hide the warnings, figure out why it's telling you them. Why go to a grocery store four times for four items when you can get all four from one visit? 
```suggestion
import {
  getCollection, deleteDevice, updateDevice, addDevice,
} from '@/firebase/techCenterCheckout.Data';
```